install-panel.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+#shellcheck source=/dev/null
```

Comment:
See: https://github.com/koalaman/shellcheck/wiki/SC1091

Reply:
Not needed, as already ignored at the project level. Check `travis.yml`.

Reply:
Ok.